beta/nncf/tensorflow/sparsity/rb/loss.py

```diff
@@ -0,0 +1,74 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import tensorflow as tf
+
+from nncf.api.compression import CompressionLoss
+from beta.nncf.tensorflow.layers.wrapper import NNCFWrapper
+from beta.nncf.tensorflow.sparsity.rb.operation import OP_NAME
+
+
+class SparseLoss(CompressionLoss):
+    def __init__(self, sparse_layers: [NNCFWrapper] = None, target=1.0, p=0.05):
+        super().__init__()
+        self._sparse_layers = sparse_layers
+        self.target = tf.Variable(target, trainable=False)
+        self.p = p
+        self.disabled = False
```

Comment:
Have to change this attribute to tf.Variable 
```suggestion
        self.disabled = tf.Variable(False)
```

Reply:
Done